src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  onClose: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  onClose,
+}) => {
+  const [user, setUser] = useState<User>();
```

Comment:
if you don't have a value yet - put `null`
```suggestion
  const [user, setUser] = useState<User | null>(null);
```